### PR TITLE
BOSA21Q1-386: add fragment caching for initiative#show view

### PR DESCRIPTION
**Pull request by aokon**
Updated:

Added a fragment caching for static fragments related to
current_initiative.

### CODE REVIEW

File: app/views/decidim/initiatives/initiatives/show.html.erb [hunk 1]
```diff
@@ -50,56 +52,62 @@ edit_link(
...
+          <h2 class="heading2" <%= "id=#{translate_helper_for(element: :title, model: current_initiative)}" %>>
+            <%= translated_attribute(current_initiative.title) %>
+          </h2>
+          <%= render partial: "author", locals: { initiative: current_initiative } %>
           <%= translate_button_helper_for(
                   title: translated_attribute(current_initiative.title),
                   body: translated_attribute(current_initiative.description),
                   model: current_initiative,
-                  options: { class: "show-for-small-only" }
+                  options: { class: "hide-for-small-only" }
```
Comment (aokon) [inline app/views/decidim/initiatives/initiatives/show.html.erb:84]: Hmm I'm not sure why this change is included, since I haven't made it 🤔

**Review by sergei-krylov — APPROVED** [review]
